torrent_tracker_scraper/scraper.py

```diff
@@ -115,9 +118,27 @@ def get_good_infohashes(self) -> list:
             )
         return good_infohashes
 
+    def get_trackers_viafile(self,trackers,filename):
+        my_file = Path(filename)
+        try:
+            my_abs_path = my_file.resolve(strict=True)
+        except FileNotFoundError as e:
+            logger.error("External tracker file not found: %s", e)
+            #raise Exception("External tracker file not found: %s" % e)
+        else:
+            file1 = open(filename, 'r')
```

Comment:
I think we can use `Path().open()` to also open and read the file so `file1 = my_file.open() `

https://docs.python.org/3/library/pathlib.html#pathlib.Path.open